tests/unit/conftest.py

```diff
@@ -1,52 +1,65 @@
 """Unit test specific config."""
+# pylint: disable=import-outside-toplevel
 from __future__ import annotations
 
 from typing import TYPE_CHECKING
 
 import pytest
-from saq.job import Job
 from starlite.datastructures import State
 from starlite.enums import ScopeType
 
-from starlite_saqlalchemy.testing import GenericMockRepository
-from tests.utils.domain.authors import Author
-from tests.utils.domain.authors import Service as AuthorService
-from tests.utils.domain.books import Book
-from tests.utils.domain.books import Service as BookService
+from starlite_saqlalchemy import constants
 
-from ..utils import controllers
+if constants.IS_SAQ_INSTALLED:
+    from saq.job import Job
 
 if TYPE_CHECKING:
 
     from starlite import Starlite
     from starlite.types import HTTPResponseBodyEvent, HTTPResponseStartEvent, HTTPScope
 
+    from starlite_saqlalchemy.testing.generic_mock_repository import (
+        GenericMockRepository,
+    )
+    from tests.utils.domain.authors import Author
+    from tests.utils.domain.books import Book
 
-@pytest.fixture(name="author_repository_type")
+
+@pytest.fixture(name="author_repository_type", autouse=constants.IS_SQLALCHEMY_INSTALLED)
```

Comment:
these shouldn't be autouse if the dep is installed, they should call `pytest.skip()` if the dep isn't installed. Have done in #257